cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/validation/admins/AdminsValidator.kt

```diff
@@ -1,13 +1,39 @@
 package com.cognifide.cogboard.config.validation.admins
 
+import com.cognifide.cogboard.config.model.Admins
 import com.cognifide.cogboard.config.validation.Validator
-import java.io.File
+import com.fasterxml.jackson.databind.JsonMappingException
+import com.fasterxml.jackson.databind.MapperFeature
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.vertx.core.logging.Logger
+import io.vertx.core.logging.LoggerFactory
 
 object AdminsValidator : Validator {
-    override fun validate(config: String): Boolean {
-        if (File("/data/admins.json").exists()) {
-            return true
+
+    private val LOGGER: Logger = LoggerFactory.getLogger(AdminsValidator::class.java)
+    private val mapper = jacksonObjectMapper().disable(MapperFeature.ALLOW_COERCION_OF_SCALARS)
```

Comment:
can we move mapper to `Validator` interface and re-use it in all validators?

Reply:
moved